advanced_html_css/responsive_design/natural_responsiveness.md

```diff
@@ -8,7 +8,7 @@ This section contains a general overview of topics that you will learn in this l
 
 1. You'll learn a few tips for keeping your sites naturally responsive.
 
-Plain HTML, with no CSS is responsive. [Make sure to read the page](https://codyloyd.github.io/responsive-html/), and shrink your browser down to the size of a phone. It works perfectly! You could read that site on an apple watch.
+Plain HTML, with no CSS is responsive. Visit this [HTML-only page with no CSS](https://codyloyd.github.io/responsive-html/) and shrink your browser down to the size of a phone. It works perfectly! You could read that site on an apple watch.
```

Comment:
```suggestion
### Not everything needs CSS

Plain HTML, with no CSS is responsive. Visit this [HTML-only page with no CSS](https://codyloyd.github.io/responsive-html/) and shrink your browser down to the size of a phone. It works perfectly! You could read that site on an apple watch.
```

To resolve the TOP003 error here regarding only having a ul and nothing following it within the `Lesson overview` section, we can put this stuff under its own section heading as above.